src/main/java/org/sagebionetworks/bridge/validators/ParticipantRosterRequestValidator.java

```diff
@@ -34,4 +41,10 @@ public void validate(Object target, Errors errors) {
             }
         }
     }
+
+    private void validatePassword(String password) {
+        Errors error = Validate.getErrorsFor(password);
+        PasswordPolicy passwordPolicy = new PasswordPolicy(8, true, false, true, true);
+        ValidatorUtils.validatePassword(error, passwordPolicy, password);
+    }
```

Comment:
This code doesn't actually do anything if the password fails to validate. You should instead pass in the errors from validate().